geopy/util.py

```diff
@@ -80,3 +80,10 @@ def decode_page(page):
         else: # requests?
             encoding = page.headers.get("charset") or "utf-8"
             return str(page.content, encoding=encoding)
+
+
+def get_version():
```

Comment:
You mean really an arbitrary constant like "1.0" or the version number of geopy? In my opinion for traceability and debugging it makes sense that it is the actual geopy package version? But in this case a constant would have to be held in sync manually...